src/Microsoft.VisualStudio.Editors/PropPages/ReferencePropPage.vb

```diff
@@ -781,6 +785,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         End Function
 
         Private Function GetCurrentImportsList() As String()
+            'TODO : Remove the below empty return after GetCurrentImportsList() is implemented 
+            Return New String() {}
+
             Dim theVSProject As VSLangProj.VSProject
             Dim _Imports As VSLangProj.Imports
```

Comment:
VsLangProj.Imports returns back as Not implemented, Need to find a way to get the imports. Any hint on which API to call in roslyn ?